charts/victoria-metrics-k8s-stack/templates/_helpers.tpl

```diff
@@ -256,9 +256,6 @@ If release name contains chart name it will be used as a full name.
     {{- $_ := set $vmAlertRemotes "notifiers" . }}
   {{- end }}
   {{- $spec := deepCopy (omit $Values.vmalert.spec "notifiers") | mergeOverwrite $vmAlertRemotes | mergeOverwrite $vmAlertTemplates | mergeOverwrite $spec }}
-  {{- if not (or (hasKey $spec "notifier") (hasKey $spec "notifiers") (hasKey $spec "notifierConfigRef")) }}
-    {{- fail "`notifier`, `notifiers` or `notifierConfigRef` should be set for vmalert"}}
-  {{- end }}
```

Comment:
Wdyt about enforcing user to set `notifier.blackhole` if there are no other notifiers set?
This allows to make sure that user explicitly wants to blackhole all notifications and did not forget to set notifier by acccident.

Reply:
updated PR